augur/tasks/data_analysis/insight_worker/tasks.py

```diff
@@ -675,7 +675,7 @@ def clear_insight(repo_id, new_score, new_metric, new_field, logger):
                     AND ri_metric = '{}'
         """.format(insight['repo_id'], insight['ri_metric'])
         try:
-            result = engine.execute(deleteSQL)
+            result = engine.connect().execute(deleteSQL)
```

Comment:
Similar to what I mentioned in another comment. I think we should use `with engine.connect()` to make sure the connection is disposed afterwards